### DISCUSSION TITLE
Welcome to hirob-em-mechanical Discussions!

### DISCUSSION THREAD

**Discussion by husamettinarabaci** [Ubn-Jr/hirob-em-mechanical#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Talha-Dogan** [comment]
What's the purpose of Hirob-em-mechanical?


Hello, I am @Talha-Dogan I am the chairman of the Hi-Rob project, I am responsible for 3D design and 3D Printing and CNC at the same time, I am here to talk about what we will do in our project "hirob-em-mechanical" The title is a title opened for the 3D outer inner case design of our project. The 3D model that will be designed in accordance with the circuit board and internal circuit elements will also be designed in accordance with the circuit board and internal circuit elements, and the material and design application is also very important. we will use SOLIDWORKS for this project.
